src/commands/synthetics/command.ts

```diff
@@ -82,8 +82,17 @@ export class RunTestCommand extends Command {
     } catch (error) {
       if (error instanceof CiError) {
         this.reportCiError(error, this.reporter)
-        if (error instanceof CriticalError && this.config.failOnCriticalErrors) {
-          return 1
+
+        if (error instanceof CriticalError) {
+          if (this.config.failOnCriticalErrors) {
+            return 1
+          } else {
+            this.reporter.error(
+              chalk.yellow(
+                'Because `failOnCriticalErrors` is not set, the command will exit with an error code 0. Use `failOnCriticalErrors: true` to exit with an error code 1.\n'
```

Comment:
`failOnCriticalErrors` may be set, but to `false`, and we would still exit with 0 code.

I guess this is nit picking: how about "is not set or disabled"?